### PR TITLE
Add install instructions to the README file

### PR DESCRIPTION
**Pull request by munckymagik**
@benlovell @mconnell

### CODE REVIEW

**Comment by mconnell** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+
+Setup the database:
+
+```sh
+$ cp spec/dummy/config/database.example.yml spec/dummy/config/database.yml
+```
+Be sure to remove or modify the `username` attribute.
+
+```sh
+$ bundle exec rake db:create \
```
Comment (benlovell) [inline README.md]: These can be bundled up together `rake db:create db:migrate db:schema:load`

**Comment by benlovell** [comment]
:+1: other than my minor comment.

File: README.md (hunk 1 above)
Comment (munckymagik) [inline README.md]: Note to reviewers: this issue was discussed [here](https://github.com/moneyadviceservice/rad/pull/181/files#r28778203) and we decided to leave as is.

**Pull request merged**